packages/dataviews/src/components/dataform/stories/index.story.tsx

```diff
@@ -99,13 +115,17 @@ export const Default = ( { type }: { type: 'panel' | 'regular' } ) => {
 		fields: [
 			'title',
 			'order',
+			{
+				id: 'status',
```

Comment:
I like how expanding the field from string to object allows us to absorb a couple of use cases.

Have a couple of thoughts:

- I haven't read the full code yet, but judging from what I tested, I presume by reusing the `id` from an actual field we are able to reuse the field `label`. I understand this may be useful if the only thing a field does is changing its layout. However, how does it look when we want to provide a different title for grouped fields? Do we provide a `label` here as well?

- Another thing I'll be looking in the code review is the logic for stacking fields horizontally or vertically. How does that work without the `direction` config?